get-pipsi.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 import os
 import sys
+from subprocess import call
```

Comment:
As said, you need to wrap this for all invocations of ``call``, and catch OSError:

- Windows raises [WindowsError](https://docs.python.org/2/library/exceptions.html?highlight=windowserror#exceptions.WindowsError) when commands are not found, which is a subclass of ``OSError``.
- Python 2 raises OSError, Python 3 raises FileNotFoundError.